src/main/qss.js

```diff
@@ -99,6 +99,7 @@ fluid.defaults("gpii.app.qssWrapper", {
             restartWarningNotification: null
         },
 
+        lastEnvironmentalLoginGpiiKey: null,
```

Comment:
I'd leave this blank, with just a comment, on the off chance that your "null" value gets propagated back into the system on startup